docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -333,8 +333,30 @@ Example::
         returned: when supported
         type: string
         sample: 2a5aeecc61dc98c4d780b14b330e3282
-    ...
+    '''
 
+    RETURN = '''
+    packages:
+        description: Information about package requirements
+        returned: On success
+        type: complex
+        contains:
+            missing:
+                description: Packages that are missing from the system
+                returned: success
+                type: list
+                sample:
+                    - libmysqlclient-dev
+                    - libxml2-dev
+            badversion:
+            description: Packages that are installed but at bad versions.
```

Comment:
I think this needs indented further, since it should be part of `badversion`

Reply:
Thanks! Fixed.